README.md

```diff
@@ -146,9 +156,9 @@ dual licensed as above, without any additional terms or conditions.
 
 This project is currently maintained by the following developers:
 
-|       Name       |      Email Address       |                GitHub Username               |
-|:----------------:|:------------------------:|:--------------------------------------------:|
-| Jasper Wiegratz  | wiegratz@uni-bremen.de   |       [@jwhb](https://github.com/jwhb)       |
-| Jan Romann       | jan.romann@uni-bremen.de |      [@JKRhb](https://github.com/JKRhb)      |
+|      Name       |      Email Address       |          GitHub Username           |
+| :-------------: | :----------------------: | :--------------------------------: |
+| Jasper Wiegratz |  wiegratz@uni-bremen.de  |  [@jwhb](https://github.com/jwhb)  |
+|   Jan Romann    | jan.romann@uni-bremen.de | [@JKRhb](https://github.com/JKRhb) |
```

Comment:
There are some unrelated changes to the formatting here and above that should probably be reverted.